core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -805,7 +805,7 @@ impl ActualStateManagerDatabase {
     }
 }
 
-impl<R: ReadableRocks> StateManagerDatabase<R> {
+impl StateManagerDatabase<DirectRocks> {
```

Comment:
I guess you can now simply pull it under `impl ActualStateManagerDatabase {`? (line 775)